__tests__/app.test.js

```diff
@@ -22,3 +22,41 @@ describe("Get Categories", () => {
 			});
 	});
 });
+
+describe("Get Reviews", () => {
+	test("Return reviews and a status code 200", () => {
+		return request(app)
+			.get("/api/reviews")
+			.expect(200)
+			.then((result) => {
+				result.body.reviews.forEach((review) => {
+					expect(review).toEqual(
+						expect.objectContaining({
+							title: expect.any(String),
+							designer: expect.any(String),
+							owner: expect.any(String),
+							review_id: expect.any(Number),
+							review_img_url: expect.any(String),
+							review_body: expect.any(String),
+							category: expect.any(String),
+							created_at: expect.any(String),
+							votes: expect.any(Number),
+							comment_count: expect.any(String),
+						})
```

Comment:
We should expect a number here. You will need to explore how to change the data type in your query